src/Init/Core.lean

```diff
@@ -575,8 +605,9 @@ theorem Ne.elim (h : a ≠ b) : a = b → False := h
 
 theorem Ne.irrefl (h : a ≠ a) : False := h rfl
 
-theorem Ne.symm (h : a ≠ b) : b ≠ a :=
-  fun h₁ => h (h₁.symm)
+theorem Ne.symm (h : a ≠ b) : b ≠ a := fun h₁ => h (h₁.symm)
+
+theorem ne_comm {α} {a b : α} : a ≠ b ↔ b ≠ a := ⟨Ne.symm, Ne.symm⟩
```

Comment:
I know the naming looks inconsistent here.  I could make a pass through to clean it up, but those changes would make the diff even bigger.

Reply:
It is inconsistent for a good reason: in the first case we can use dot notation, and this is done frequently with `Ne`.